node/cmd/guardiand/adminclient.go

```diff
@@ -549,8 +549,20 @@ func runChainGovernorResetReleaseTimer(cmd *cobra.Command, args []string) {
 	}
 	defer conn.Close()
 
+	var numDays = uint32(1)
+	if len(args) > 1 {
+		numDaysArg, err := strconv.Atoi(args[1])
+
+		if err != nil {
+			log.Fatalf("invalid num_days: %v", err)
+		}
+
+		numDays = min(max(uint32(numDaysArg), 1), 7)
```

Comment:
I'm not sure if forcing into a valid range is what we want to do here? Would it not be better to instead return an error if num days <1 or >7? Especially if the description is updated

Reply:
I agree. Makes sense for it to be more intentional.

Reply:
I deleted this logic and just set `numDays` to `uint32(numDaysArg)`. The range check already exists in the server-side handler in `adminserver.go`.